webpack.config.js

```diff
@@ -186,15 +185,16 @@ const generateConfig = ({extensionPath, devMode=false, customOutputPath, analyze
     entry,
     output: {
       path: outputPath,
-      filename: `auspice.bundle${!devMode ? ".[contenthash]" : ""}.js`,
+      filename: `auspice.[name].bundle${!devMode ? ".[contenthash]" : ""}.js`,
       chunkFilename: `auspice.chunk.[name].bundle${!devMode ? ".[chunkhash]" : ""}.js`,
       publicPath: "/dist/"
     },
     resolve: {
-      alias: aliasesToResolve
-    },
-    node: {
-      fs: 'empty'
+      alias: aliasesToResolve,
```

Comment:
Something (perhaps this line?) is resulting in auspice.us not building coreectly. I tested this in the auspice.us repo by running `npm ci && npm install nextstrain/auspice#7be662dc4554319a3d3f02fc5d75cdf8ea5366d8` 

1. `npm run build` works, although it has a couple of deprecation warnings (I _think_ these are new)

```
WARNING in ../../auspice_client_customisation/splash.js 115:164-171
Should not import the named export 'version' (imported as 'version') from default-exporting module (only default export is available soon)
...
WARNING in ../../auspice_client_customisation/splash.js 115:194-214
Should not import the named export 'dependencies'.'auspice' (imported as 'dependencies') from default-exporting module (only default export is available soon)
```

2. Loading the app (`npm run start`) shows that the extensions didn't work -- the splash page is that of default auspice not auspice.us. I'm surprised there weren't build errors.

3. I tried to mimic https://github.com/nextstrain/nextstrain.org/pull/634, but auspice.us uses the `@auspice` webpack alias so it's not quite the same. Didn't work. Here's what I tried:

```diff
diff --git a/auspice_client_customisation/handleDroppedFiles.js b/auspice_client_customisation/handleDroppedFiles.js
index 96a93fc..b8a5e3e 100644
--- a/auspice_client_customisation/handleDroppedFiles.js
+++ b/auspice_client_customisation/handleDroppedFiles.js
@@ -1,10 +1,10 @@
-import { createStateFromQueryOrJSONs } from "@auspice/actions/recomputeReduxState";
-import { errorNotification, warningNotification } from "@auspice/actions/notifications";
-import { Dataset, addEndOfNarrativeBlock, getDatasetNamesFromUrl } from "@auspice/actions/loadData";
-import { parseMarkdownNarrativeFile } from "@auspice/util/parseNarrative";
-import { parseMarkdown } from "@auspice/util/parseMarkdown";
-import { isAcceptedFileType as isAuspiceAcceptedFileType } from "@auspice/actions/filesDropped/constants";
-import newickToAuspiceJson from "./parseNewick";
+import { createStateFromQueryOrJSONs } from "@auspice/actions/recomputeReduxState.js";
+import { errorNotification, warningNotification } from "@auspice/actions/notifications.js";
+import { Dataset, addEndOfNarrativeBlock, getDatasetNamesFromUrl } from "@auspice/actions/loadData.js";
+import { parseMarkdownNarrativeFile } from "@auspice/util/parseNarrative.js";
+import { parseMarkdown } from "@auspice/util/parseMarkdown.js";
+import { isAcceptedFileType as isAuspiceAcceptedFileType } from "@auspice/actions/filesDropped/constants.js";
+import newickToAuspiceJson from "./parseNewick.js";
 
 /* The following requires knowledge of how auspice works, is undocumented, and is liable to change since auspice
 doesn't officially expose these functions */
diff --git a/auspice_client_customisation/splash.js b/auspice_client_customisation/splash.js
index b06c118..73cf222 100644
--- a/auspice_client_customisation/splash.js
+++ b/auspice_client_customisation/splash.js
@@ -1,6 +1,6 @@
 import React from "react"; // eslint-disable-line
-import { handleDroppedFiles } from "./handleDroppedFiles";
-import { P, Bold, Title, NextstrainTitle, CenterContent, Line, GitHub } from './styles';
+import { handleDroppedFiles } from "./handleDroppedFiles.js";
+import { P, Bold, Title, NextstrainTitle, CenterContent, Line, GitHub } from './styles.js';
 import { version, dependencies } from "../package.json";
```

Reply:
Fixed by adding 5 characters 🤦 (see [force-push](https://github.com/nextstrain/auspice/compare/33b5cbd7eb9a1c1ad73b10071b8c1c79640deeb9..1d2825d87b29476b4abcd6a242276a87a1b88399)).

After an afternoon of tinkering and many red herrings (which at least helped me understand Webpack a little better), I figured this out. Turns out that, without that change, `auspice view` thinks that it is running from Auspice's root directory, not auspice.us, and therefore serves `auspice/dist` instead of `auspice.us/dist`.

Now I'm asking myself, how did it work for nextstrain.org? My guess is that nextstrain.org doesn't use `auspice view` and instead serves the `auspice-client` files directly ([here](https://github.com/nextstrain/nextstrain.org/blob/8ff7529de28e81f6e56e6dbc08a8cfe4974af436/src/endpoints/static.js#L17-L18)). Does that sound right?

Reply:
Also, good to note that this was already observable in the auspice.us review apps https://github.com/nextstrain/auspice.us/pull/43 and https://github.com/nextstrain/auspice.us/pull/44:

![image](https://user-images.githubusercontent.com/13424970/205419763-3f7f66ad-a711-4022-9092-bd10a3d02ff1.png)

So once those re-deploy with the new changes, I'd expect to see them working properly.

Reply:
Re: deprecation warnings, I've created https://github.com/nextstrain/auspice.us/pull/45 to address that.

Reply:
> My guess is that nextstrain.org doesn't use auspice view and instead serves the auspice-client files directly

Correct!

I'll re-test this today. Future to-do:  improve the message "Serving auspice version ${version}" as if that indicated it was using the non-auspice.us version it probably would have quickened the search for this bug.

Reply:
> Future to-do

Done with #1608.

This is the output of `npm run start` from 33b5cbd7eb9a1c1ad73b10071b8c1c79640deeb9 (pre-fix):

```
---------------------------------------------------
Auspice server now running at http://localhost:4000
Serving auspice version 2.41.0
Custom server handlers provided.
---------------------------------------------------
```

This is the output after applying changes from #1608 (it's an error so it does not actually run the server):

```
ERROR: Auspice build files not found under /Users/vlin/Dropbox/tmp/auspice.us. Did you run `auspice build` in this directory?
```

Reply:
1d2825d fixes this 🎉